src/components/Activities/Activity3/Animations/PlantSprout.tsx

```diff
@@ -0,0 +1,57 @@
+import anime, { AnimeTimelineInstance } from 'animejs';
+import React, { useContext, useEffect, useRef } from 'react';
+import { CarouselContext } from '../../../shared/Carousel';
+import ScalingSlide from '../../../shared/ScalingSlide';
+
+interface ManyEmployeesProps {
+  start: boolean,
+}
+
+function ManyEmployees(props: ManyEmployeesProps): JSX.Element {
+  const { reloadTime } = useContext(CarouselContext);
+  const timeline = useRef<AnimeTimelineInstance | null>(null);
+
+  useEffect(() => {
+    timeline.current = anime.timeline({
+      autoplay: false,
+      easing: 'easeInOutSine',
+    });
+
+    timeline.current?.add({
+      targets: '#plant-sprout',
+      translateY: [0, -190],
+      duration: 1000,
+    }).add({
+      targets: '#left-leaf',
```

Comment:
no need to change, but for future reference, you can do this
```suggestion
      targets: ['#left-leaf', bleb, bleb],
```